R/geom-bar.r

```diff
@@ -123,16 +129,18 @@ GeomBar <- ggproto("GeomBar", GeomRect,
     params
   },
 
-  extra_params = c("na.rm", "orientation"),
+  extra_params = c("just", "na.rm", "orientation"),
 
   setup_data = function(data, params) {
     data$flipped_aes <- params$flipped_aes
     data <- flip_data(data, params$flipped_aes)
     data$width <- data$width %||%
       params$width %||% (resolution(data$x, FALSE) * 0.9)
+    data$just <- data$just %||% params$just %||% 0.5
```

Comment:
```suggestion
    data$just <- params$just %||% 0.5
```

In `width`'s case, `width` might be provided by the `Stat`; e.g. the bin widths of a histogram can vary so the computed data contains the `width`s. But, there's no Stat that returns `just`.

Reply:
Makes sense, thanks!